src/components/AppLayout/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { theme as styledTheme } from '@gnosis.pm/safe-react-components'
+import { fireEvent, render, screen } from '@testing-library/react'
+import { history, store } from 'src/store'
+import theme from 'src/theme/mui'
+import Providers from 'src/components/Providers'
+import Footer from './index'
+import CookiesBanner from 'src/components/CookiesBanner'
+
+describe('<Footer>', () => {
+  it('Should render Footer component', () => {
+    const { container } = render(
+      <Providers store={store} history={history} styledTheme={styledTheme} muiTheme={theme}>
```

Comment:
Is there any way to set this up once for all tests in the project?
So that you don't have to wrap components with the provider in each `it`?

Reply:
there's a good example in testing library docs: https://testing-library.com/docs/react-testing-library/setup#custom-render